src/order/boolean_algebra.lean

```diff
@@ -754,6 +754,14 @@ end of_core
 section boolean_algebra
 variables [boolean_algebra α]
 
+--TODO@Yaël: Once we have co-Heyting algebras, we won't need to go through `boolean_algebra.of_core`
```

Comment:
Let's not leave names attached to TODOs, unless you have a very concrete immediate plan to do it in a follow-up PR. Otherwise it's just cleanup work and confusion later.

Reply:
I have such a plan :)